kibana.md

```diff
@@ -1101,3 +1101,187 @@ an admin, who has just configured access for a new user. They would like
+| metadata.lastSessionActivityDate     | Date or undefined                                                      | 2023-03-23T19:50:37.932Z                                                     | Date of the last session activity. Using in the context of a session timeout         |
+| metadata.authorizationHeaders        | Map<string, string>                                                    | Map(2) {'authorization' => 'Basic BWRtaW46ZGV2', 'cookie' => 'cookie value'} | The map of all headers use during the authorization                                  |
+| metadata.username                    | string                                                                 | 'John'                                                                       | User name defined in the ACL                                                         |
+| metadata.kibanaHiddenApps            | string[]                                                               | [ 'Enterprise Search, Overview', 'Observability' ]                           | List of the hidden Apps defined in the ACL                                           |
+| metadata.currentGroup                | string                                                                 | 'administrators'                                                             | Currently selected group by the user                                                 |
+| metadata.availableGroups             | string[]                                                               | [ 'administrators', 'infosec', 'template' ],                                 | List of the all available groups                                                     |
+| metadata.kibanaAccess                | 'ro_strict' or 'ro' or 'rw' or 'admin' or 'unrestricted' or 'api_only' | 'admin'                                                                      | User kibana access                                                                   |
+| metadata.kibanaIndex                 | string or undefined                                                    | '.kibana_administrators'                                                     | Currently selected group kibana index defined in the ACL                             |
+| metadata.kibanaTemplateIndex         | string or undefined                                                    | '.kibana_template'                                                           | Kibana template index defined in the kibana.yml readonlyrest_kbn.kibanaTemplateIndex |
+| metadata.origin                      | string or undefined                                                    |                                                                              |                                                                                      |
```

Comment:
@sscarduzio @coutoPL  I have a question about the origin which is an `x-ror-origin` what's the function and example of this value ( i didn't find any information in the es/kibana plugin code and docs, and this value is always undefined)

Reply:
this is how it's described in the swagger:
```
  x-ror-origin:
          type: string
          description: the value taken from JWT's x-ror-origin claim
```

"... when `ror_kbn_auth` is matched" (should be added)

I made this description based on ROR ES code.

Reply:
Thanks, I forgot that swagger is not a part of the `elasticsearch-readonlyrest-plugin` repository that's why I couldn't find any description :D

Reply:
IIRC the x-ror-origin value should take the name of the OIDC/SAML connector, and is used in Kibana to resolve to which "single sign out" URL the browser should be redirected to. 

Imagine you have 2 SAML IdP configured in kibana.yml, the user hits logout, we delete their cookie, and immediately redirect to the logout URL of the IdP. But which one of the two? :)

Reply:
@Dzuming it's but I've added the description recently and it's in develop (not in master yet)